#!/usr/bin/env python
PACKAGE = "aiv_controller_lqr"

from dynamic_reconfigure.parameter_generator_catkin import *


gen = ParameterGenerator()

gen.add("th_dot_goal_val", double_t, 0, "th_dot_goal_value", 146.635, 0, 1000)
gen.add("th_dot_d_val", double_t, 0, "th_dot_d_value", 0.1842, 0, 1000)
gen.add("x_dot_goal_val", double_t, 0, "x_dot_goal_value", 147.16, 0, 1000)
gen.add("x_dot_d_val", double_t, 0, "x_dot_d_value", 0.1842, 0, 1000)
gen.add("y_dot_goal_val", double_t, 0, "y_dot_goal_value", 147.16, 0, 1000)
gen.add("y_dot_d_val", double_t, 0, "y_dot_d_value", 0.1842, 0, 1000)

exit(gen.generate(PACKAGE, "aiv_controller_lqr", "controllerServer"))
